common/app/model/content.scala

```diff
@@ -364,17 +376,12 @@ object Content {
       wordCount = Jsoup.clean(fields.body, Whitelist.none()).split("\\s+").length,
       showByline = fapiutils.ResolvedMetaData.fromContentAndTrailMetaData(apiContent, TrailMetaData.empty, cardStyle).showByline,
       hasStoryPackage = apifields.flatMap(_.hasStoryPackage).getOrElse(false),
-      rawOpenGraphImage = {
-        val bestOpenGraphImage = if (FacebookShareUseTrailPicFirstSwitch.isSwitchedOn) {
-          trail.trailPicture.flatMap(_.largestImageUrl)
-        } else {
-          None
-        }
-        bestOpenGraphImage
-          .orElse(elements.mainPicture.flatMap(_.images.largestImageUrl))
-          .orElse(trail.trailPicture.flatMap(_.largestImageUrl))
-          .getOrElse(Configuration.images.fallbackLogo)
-      }
+      rawOpenGraphImage =
+        FacebookShareUseTrailPicFirstSwitch.isSwitchedOn
+          .toOption(trail.trailPicture.flatMap(_.largestImage))
+          .flatten
+          .orElse(elements.mainPicture.flatMap(_.images.largestImage))
+          .orElse(trail.trailPicture.flatMap(_.largestImage))
```

Comment:
it's just been moved into the case class instead on line 110.  I wanted the `rawOpenGraphImage` to be an `ImageAsset` so I had access to its dimensions whereas the fallback is just a string.